bedrock/firefox/tests/test_base.py

```diff
@@ -387,6 +387,39 @@ def test_yahoo_retention_funnelcake_other_locales(self, render_mock):
         template = render_mock.call_args[0][1]
         eq_(template, ['firefox/firstrun/firstrun-horizon.html'])
 
+    # New Firstrun Wow-moment funnelcake tests
```

Comment:
Could we please add another test here just to make sure the control page is shown for f=98? thanks